components/workouts/AllWorkouts.jsx

```diff
@@ -11,29 +13,100 @@ const Title = styled.Text`
   margin: 15px 5%;
 `;
 
-const AllWorkouts = (props) => {
-  const { items } = props;
+const SectionHeader = styled.Text`
+  font-family: 'Montserrat_600SemiBold';
+  font-size: 20px;
+  margin-bottom: 15px;
+  padding-left: 5%;
+  background-color: rgb(242, 242, 242);
+`;
+
+const WorkoutCardPair = styled.View`
+  flex-direction: row;
+  justify-content: space-evenly;
+  margin-bottom: 10px;
+`;
+
+const parseItems = (items) => {
+  // Sort names alphabetically
+  items.sort((a, b) => a.name.localeCompare(b.name));
+
+  // Group by first letter of each name
+  const bucketData = items.reduce((accumulator, item) => {
+    const bucket = item.name[0].toUpperCase();
+
+    // If this is the first time we've seen this letter, create a bucket
+    if (!accumulator[bucket]) {
+      accumulator[bucket] = [item];
+    } else {
+      accumulator[bucket].push(item);
+    }
+
+    return accumulator;
+  }, {});
+
+  // Pair up each bucket of data
+  const pairedBucketData = _.mapValues(bucketData, (data) => {
+    const pairData = [];
+
+    // If we go "out of bounds" here it will just make right undefined
+    for (let i = 0; i < data.length; i += 2) {
+      pairData.push({ left: data[i], right: data[i + 1] });
+    }
+    return pairData;
+  });
 
-  const renderCard = ({ item }) => (
+  return pairedBucketData;
+};
+
+const renderCard = ({ item: { left, right } }) => (
+  <WorkoutCardPair>
     <WorkoutCard
-      name={item.name}
-      subtext={item.subtext}
-      displayEllipses={item.displayEllipses}
-      onIconPress={item.onIconPress}
-      onPress={item.onPress}
-      color={item.color}
+      name={left.name}
+      subtext={left.subtext}
+      displayEllipses={left.displayEllipses}
+      deleteWorkout={left.deleteWorkout}
+      onPress={left.onPress}
+      color={left.color}
+      key={left.name + left.subtext}
     />
-  );
+    {right ? (
+      <WorkoutCard
+        name={right.name}
+        subtext={right.subtext}
+        displayEllipses={right.displayEllipses}
+        deleteWorkout={right.deleteWorkout}
+        onPress={right.onPress}
+        color={right.color}
+        key={right.name + right.subtext}
+      />
```

Comment:
I don't have a problem with the "left cards & right cards" way. It's pretty easy to understand and it works as expected. 
That being said, why did you do it like this as opposed to just using styling and numColumns?

Reply:
Unless I missed something in the documentation, there isn't numColumns for the alphabet list component I used for this. It also wasn't there for any of the other options I looked at, although documentation wasn't always great.